### ISSUE TITLE
In MIG, the relationship between GRACT and DCGM_FI_PROF_GR_ENGINE_ACTIVE ?

### ISSUE THREAD

**Issue by Lzhang-hub** [NVIDIA/dcgm-exporter#53]
## env
GPU: A100
MIG: 7g.40gb
DCGM: v2.3.2
DCGM-EXPORT: v2.3.4-2.6.4

## result
1、cmd: `dcgmi dmon -e 1001,1004 -g 2`
![image](https://user-images.githubusercontent.com/57925599/158568390-d467a8f3-b7a2-46c8-a310-a08b6415f3af.png)
2、dcgm-export
![image](https://user-images.githubusercontent.com/57925599/158568611-05f05cfe-04c8-471e-a00b-afe0c16a5587.png)
![image](https://user-images.githubusercontent.com/57925599/158568676-c94a1b49-08c2-4d1f-a152-03eef5e29558.png)
 ## question
I want to know the **GRACT** get from `dcgmi dmon` is the same as **DCGM_FI_PROF_GR_ENGINE_ACTIVE**  get from `dcgm-export`?

**Comment by nikkon-dev** [comment]
@Lzhang-hub,

Yes, the DCGM_FI_PROF_GR_ENGINE_ACTIVE in the dcgm-exporter and `dcgmi dmon -e 1001` are the same metric.
The difference is the moment when this metric is gathered and how often. 
You could try to run `dcgmi dmon -e 1001 -g 2 -d 30000` to read the value every 30 seconds (IIRC that's the dcgm-exporter default interval). Those values are not aggregated during the monitoring interval, those are instantaneous values.

**Comment by Lzhang-hub** [comment]
@nikkon-dev 
thank you replay! 
And another question, I also want monitor memory utile for MIG instances , but I can not find some metrics from dcgm-export ,can you give some advice, thanks!

**Comment by Lzhang-hub** [comment]
@nikkon-dev 
We test mig with a train job and monitor through dcgm-exporter+prometheus+grafana, But we got the error result:

First,got mig instance for job through nvidia-smi: GPU0 GI 4
![image](https://user-images.githubusercontent.com/57925599/161364432-1f993fd9-cdeb-4aec-a13e-e8243654d155.png)

Then, got gpu utile from grafana, we found all GPU are used:
GI 3
![image](https://user-images.githubusercontent.com/57925599/161364525-1a0663ef-1caf-43b6-a246-58de96e10960.png)
GI 5
![image](https://user-images.githubusercontent.com/57925599/161364567-b94f262e-4f27-453f-a52d-d685ee3ab2be.png)

We make sure that only one task runs! 
Can you give some advice? Thanks!
